src/Microsoft.DocAsCode.Build.Engine/DocumentBuilder.cs

```diff
@@ -30,15 +28,15 @@ public class DocumentBuilder : IDisposable
         internal IEnumerable<IInputMetadataValidator> MetadataValidators { get; set; }
 
         private readonly string _intermediateFolder;
-        private readonly List<PostProcessor> _postProcessors = new List<PostProcessor>();
         private readonly CompositionHost _container;
         private readonly BuildInfo _currentBuildInfo =
             new BuildInfo
             {
                 BuildStartTime = DateTime.UtcNow,
                 DocfxVersion = typeof(DocumentBuilder).Assembly.GetCustomAttribute<AssemblyFileVersionAttribute>()?.Version
             };
-        private BuildInfo _lastBuildInfo;
+        private readonly BuildInfo _lastBuildInfo;
+        private readonly HandlePostProcessors _handlePostProcessors;
```

Comment:
how about PostpcessorsBuilder...

---
In reply to: [93848200](https://github.com/dotnet/docfx/pull/1055#discussion_r93848200) [](ancestors = 93848200)